.stylelintrc.json

```diff
@@ -0,0 +1,96 @@
+{
+  "plugins": [
+    "stylelint-scss"
+  ],
+  "rules": {
+    "declaration-no-important": true,
+    "media-feature-name-no-vendor-prefix": null,
```

Comment:
could you create JIRAs for those rules have been disabled? 
(probably under epic https://oktainc.atlassian.net/browse/OKTA-196695)

Reply:
Yepp already have one! Will add in the epic ^^

Reply:
Added https://oktainc.atlassian.net/browse/OKTA-198163 to the epic

Reply:
@manueltanzi-okta is this what that ticket is referring to: https://github.com/okta/okta-ui/blob/master/packages/nim/.stylelintrc.json ?

Reply:
@edburyenegren-okta yepp!